device/transport/mqtt/devdoc/mqtt_requirements.md

```diff
@@ -96,24 +109,38 @@ The `sendEvent` method sends an event to an IoT hub on behalf of the device indi
 
 **SRS_NODE_COMMON_MQTT_BASE_16_015: [** The `sendEvent` method shall serialize the `expiryTimeUtc` property of the message as a key-value pair on the topic with the key `$.exp`. **]**
 
+**SRS_NODE_DEVICE_MQTT_16_023: [** The `sendEvent` method shall connect the Mqtt connection if it is disconnected. **]**
+
+**SRS_NODE_DEVICE_MQTT_16_024: [** The `sendEvent` method shall call its callback with an `Error` that has been translated using the `translateError` method if the `MqttBase` object fails to establish a connection. **]**
+
+**SRS_NODE_DEVICE_MQTT_16_025: [** The `sendEvent` method shall be deferred until either disconnected or connected if it is called while `MqttBase` is establishing the connection. **]**
```

Comment:
>deferred [](start = 67, length = 8)

I think I commented on a different CR, but I don 't like "deferred" here because that talks about internal implementation rather than behavior.  
Maybe:
If sendEvent is called while MqttBase is establishing the connection, it shall want until the connection is established and then send the event.
If sendEvent is called while MqttBase is establishing the connection, and MqttBase fails to establish the connection, then sendEvent shall fail.